packages/logging/src/v2/config_service_v2_client.js

```diff
@@ -100,27 +100,94 @@ function ConfigServiceV2Client(gaxGrpc, grpcClients, opts) {
   var self = this;
 
   this.auth = gaxGrpc.auth;
-  var configServiceV2Stub = gaxGrpc.createStub(
-      grpcClients.google.logging.v2.ConfigServiceV2,
-      opts);
+
   var configServiceV2StubMethods = [
     'listSinks',
     'getSink',
     'createSink',
     'updateSink',
     'deleteSink'
   ];
+
+  var configServiceV2Stub;
+
   configServiceV2StubMethods.forEach(function(methodName) {
-    self['_' + methodName] = gax.createApiCall(
-      configServiceV2Stub.then(function(configServiceV2Stub) {
-        return function() {
-          var args = Array.prototype.slice.call(arguments, 0);
-          return configServiceV2Stub[methodName].apply(configServiceV2Stub, args);
-        };
-      }),
-      defaults[methodName],
-      PAGE_DESCRIPTORS[methodName]);
+    self['_' + methodName] = function() {
```

Comment:
@landrito this is where a big part of the changes begin. The function waits until it is called to do the `gax.createApiCall` behavior.